packages/next/src/build/webpack/loaders/next-flight-css-loader.ts

```diff
@@ -22,6 +22,10 @@ const NextServerCSSLoader = function (this: any, content: string) {
 
   // Only add the checksum during development.
   if (process.env.NODE_ENV !== 'production') {
+    // Server CSS modules are always available for HMR, so we attach
+    // `module.hot.accept()` to the generated module.
+    const hmrCode = 'if (module.hot) { module.hot.accept() }'
```

Comment:
That's not true for CSS Modules since the exports might change when adding new class names, etc.

Reply:
Oops this is actually only needed for global CSS, udpated!